### ISSUE TITLE
Hyundai/EU user profile changes

### ISSUE THREAD

**Issue by mikelauritsen** [Hyundai-Kia-Connect/kia_uvo#360]
Hi,
I am using this integration with a Hyundai IONIQ5.
Everything works fine, except that the user profile in the car is changed to the Bluelink user registered with the integration, every time it updates data from the car.
This doesn't happen when using the Bluelink app.
Is it possible to avoid user profile change, or is this just the way the integration works?

**Comment by cdnninja** [comment]
Would need sniffing of the traffic and comparing to figure this one out. Ideally have a dev from EU chime in on it.

**Comment by cdnninja** [comment]
Re-reading this when you say user profile do you mean language?  If so please try the "master" branch in HACS and let me know if it is solved.  It will need home assistant set to a language that the app supports.

**Comment by mikelauritsen** [comment]
No, with user profile I mean the selected Bluelink profile in the car that
holds settings for seat and mirror positions, phone connection etc.

ons. 21. des. 2022, 00:27 skrev cdnninja ***@***.***>:

**Comment by Bubo08** [comment]
Could you please clarify, where you see this profile information. If you have a debug log this would also help.
